network/framework/src/application/interface.rs

```diff
@@ -175,7 +175,7 @@ impl<Message: NetworkMessageTrait + Clone> NetworkClient<Message> {
         // We only periodically log any unavailable peers (to prevent log spamming)
         if !peers_without_a_protocol.is_empty() {
             sample!(
-                SampleRate::Duration(Duration::from_secs(10)),
+                SampleRate::Duration(Duration::from_secs(30)),
```

Comment:
Is 10 seconds really too much? If we're going to every 30 seconds, part of me feels like we should just downgrade the log to `info` or something else.